script/state.py

```diff
@@ -15,8 +16,8 @@
 states_data = cur.fetchall()
 conn.close()
 
-if len(states_data)==0:
-    print("No data fetched from database ")
+if not states_data :
+    logging.ERROR("No data fetched from database ")
```

Comment:
Test this scenario by following steps:
1. In the geovisualizer_data.sql file remove all the Insert statements for state i.e. line 505 onwards
2. Run docker compose down --volumes to remove existing db containers and data
3. docker compose build
4. make run-state

See if you get any errors, and fix them if you find some.